airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java

```diff
@@ -38,41 +41,53 @@ public OAuthHandler(ConfigRepository configRepository) {
   public OAuthConsentRead getSourceOAuthConsent(SourceOauthConsentRequest sourceDefinitionIdRequestBody)
       throws JsonValidationException, ConfigNotFoundException, IOException {
     final OAuthFlowImplementation oAuthFlowImplementation = getSourceOAuthFlowImplementation(sourceDefinitionIdRequestBody.getSourceDefinitionId());
-    return new OAuthConsentRead().consentUrl(oAuthFlowImplementation.getSourceConsentUrl(
+    final ImmutableMap<String, Object> metadata = generateSourceMetadata(sourceDefinitionIdRequestBody.getSourceDefinitionId());
+    final OAuthConsentRead result = new OAuthConsentRead().consentUrl(oAuthFlowImplementation.getSourceConsentUrl(
         sourceDefinitionIdRequestBody.getWorkspaceId(),
         sourceDefinitionIdRequestBody.getSourceDefinitionId(),
         sourceDefinitionIdRequestBody.getRedirectUrl()));
+    TrackingClientSingleton.get().track(sourceDefinitionIdRequestBody.getWorkspaceId(), "OAuth Consent - Backend", metadata);
+    return result;
   }
 
   public OAuthConsentRead getDestinationOAuthConsent(DestinationOauthConsentRequest destinationDefinitionIdRequestBody)
       throws JsonValidationException, ConfigNotFoundException, IOException {
     final OAuthFlowImplementation oAuthFlowImplementation =
         getDestinationOAuthFlowImplementation(destinationDefinitionIdRequestBody.getDestinationDefinitionId());
-    return new OAuthConsentRead().consentUrl(oAuthFlowImplementation.getDestinationConsentUrl(
+    final ImmutableMap<String, Object> metadata = generateDestinationMetadata(destinationDefinitionIdRequestBody.getDestinationDefinitionId());
+    final OAuthConsentRead result = new OAuthConsentRead().consentUrl(oAuthFlowImplementation.getDestinationConsentUrl(
         destinationDefinitionIdRequestBody.getWorkspaceId(),
         destinationDefinitionIdRequestBody.getDestinationDefinitionId(),
         destinationDefinitionIdRequestBody.getRedirectUrl()));
+    TrackingClientSingleton.get().track(destinationDefinitionIdRequestBody.getWorkspaceId(), "OAuth Consent - Backend", metadata);
```

Comment:
Just a suggestion on the event name
```suggestion
    TrackingClientSingleton.get().track(destinationDefinitionIdRequestBody.getWorkspaceId(), "Get Oauth Consent URL - Backend", metadata);
```